PlateSpace/ETL/createIndexes.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
i think we don't need this file anymore because `index.js` is now creating the indexes?

If there's other indexes in `platespaceIndexes.js` that need to be created, though (like the unique index on `reviewsRatings` - then we should just include them in `index.js` instead of adding back this file, so we aren't setting up indexes in two places.